### ISSUE TITLE
This plugin slows down Homebridge

### ISSUE THREAD

**Issue by bjornand1** [metbosch/homebridge-http-temperature#32]
Hi, thanks for this plugin, which does work, allowing me get the temperature from a Shelly 1 with Temp Add-on into Homebridge. However, I constantly see this warning:

[7/22/2022, 7:48:51 AM] [homebridge-http-temperature] This plugin slows down Homebridge. The read handler for the characteristic 'Current Temperature' was slow to respond!

Is there any way to avoid this?

**Comment by metbosch** [comment]
Does your endpoint take a long time to respond?

**Comment by bjornand1** [comment]
@metbosch No, not really, it is a Shelly 1 with temp add-on and in the Shelly app is updates quite fast.

**Comment by Lling** [comment]
I have the same problem at the moment. Im using a dht11 sensor for humidity and temperature. The humidity works but the temperature not. 

Problem fixed with a higher timeout time.
